pyproject.toml

```diff
@@ -64,8 +68,11 @@ publishing = [
     "build",
 ]
 
+[project.scripts]
+gemdash= "gemdat.dashboard:gemdash"
+
 [tool.setuptools]
-package-dir = {"gemdat" = "src", "pymatgen.analysis.gemdat" = "src"}
+package-dir = {"gemdat" = "src", "pymatgen.analysis.gemdat" = "src", "gemdat.dashboard" = "dashboard" }
```

Comment:
First, I don't think this should be a module. Second, why is this code not in `src/dashboard`?

Reply:
The benefit of a module here would be that the dashboard is quite clearly separated from the code, this makes it easy to package it separately, which in turn makes it easy for a user to only use gemdat, if they are not interested in gemdash and its dependencies.

But doing it the way of extra dependencies in the package itself is possible as well so the tradeoff here is:
- `pip install gemdat[gemdash]` to enable `gemdash`
Or support:
-  `pip install gemdat` for only gemdat
- ` pip install gemdash` which transparently pulls `gemdat` if not installed

I think I prefer the second option, but probably the first option would be less maintenance :sweat_smile:

Reply:
Ok I will go with the first option for now, makes more sense in this stage

Reply:
We could have `src/gemdat` and `src/gemdash` for separation as well. 

I think option 2 would require the dashboard to be in a separate repo. I think the benefit of this is that it can be installed separately, but also that we can manage the dependency on `gemdat` better. I.e. updating gemdat does not immediately break gemdash if we pin the version.